data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/PlainConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
```

Comment:
This is pretty much same as PlainTextConfig

Reply:
Yes , we have incorporated the changes.